python/datasources/ahr.py

```diff
@@ -0,0 +1,305 @@
+                output_row[std_col.RACE_CATEGORY_ID_COL] = RACE_GROUPS_TO_STANDARD[breakdown_value]
+
+            else:
+                output_row[breakdown] = breakdown_value
+
+            for determinant, prefix in AHR_DETERMINANTS.items():
+                per_100k_col_name = std_col.generate_column_name(prefix,
+                                                                 std_col.PER_100K_SUFFIX)
+                pct_share_col_name = std_col.generate_column_name(prefix,
+                                                                  std_col.PCT_SHARE_SUFFIX)
+
+                # replace extra space to match 65+ column correctly
+                df.replace('Voter Participation (Presidential) - Ages 65+ ',
+                           'Voter Participation (Presidential) - Ages 65+', inplace=True)
+
+                matched_row = get_matched_row(df,
+                                              state,
+                                              determinant,
+                                              breakdown_value,
+                                              breakdown)
+
+                if len(matched_row) > 0:
+                    output_row[pct_share_col_name] = matched_row['CaseShare'].values[0]
+
+                    if determinant in PER100K_DETERMINANTS:
+                        output_row[per_100k_col_name] = matched_row['Value'].values[0]
+                    else:
+                        output_row[per_100k_col_name] = matched_row['Value'].values[0] * 1000
+
+            output.append(output_row)
+
+    output_df = pd.DataFrame(output)
+
+    return output_df
+
+
+def post_process(breakdown_df: pd.DataFrame, breakdown: SEX_RACE_ETH_AGE_TYPE, geo: GEO_TYPE):
+    """
+    Merges the state IDs with population data and performs necessary calculations
+    to create a processed dataframe ready for the frontend. If the given breakdown
+    is RACE_OR_HISPANIC_COL, additional columns are added to the dataframe based on
+    the RACE_CATEGORY_ID_COL column.
+
+    Args:
+        breakdown_df: A pandas DataFrame containing all the raw AHR data.
+        breakdown: string equal to race_and_ethnicity, sex, or age.
+
+    Returns:
+        A pandas DataFrame with processed data ready for use in the frontend.
+    """
+    if breakdown == std_col.RACE_OR_HISPANIC_COL:
+        std_col.add_race_columns_from_category_id(breakdown_df)
+
+    breakdown_df = merge_state_ids(breakdown_df)
+
+    breakdown_name = 'race' if breakdown == std_col.RACE_OR_HISPANIC_COL else breakdown
+
+    breakdown_df = merge_pop_numbers(breakdown_df, cast(
+        SEX_RACE_AGE_TYPE, breakdown_name), geo)
+
+    breakdown_df = breakdown_df.rename(
+        columns={std_col.POPULATION_PCT_COL: std_col.BRFSS_POPULATION_PCT})
+
+    breakdown_df[std_col.BRFSS_POPULATION_PCT] = breakdown_df[std_col.BRFSS_POPULATION_PCT].astype(
+        float)
+    breakdown_df = breakdown_df.drop(columns=std_col.POPULATION_COL)
```

Comment:
let's change this to `std_col.AHR_POPULATION_PCT` and have everything front and back switched over to using `AHR`